### PR TITLE
July launch PR

### PR DESCRIPTION
**Pull request by CodeTorso**
I hate merge errors

### CODE REVIEW

**Comment by Dhravya** [comment]
i have it set up locally, tell me when ready to merge

File: apps/web/wrangler.toml [hunk 1]
```diff
@@ -12,8 +12,8 @@ bucket_name = "dev-r2-anycontext"
 
 [[d1_databases]]
 binding = "DATABASE"
-database_name = "dev-d1-anycontext"
-database_id = "fc562605-157a-4f60-b439-2a24ffed5b4c"
+database_name = "prod-d1-tutorial"
+database_id = "325fe40a-bdfc-43ec-837b-021bbcc570e8"
```
Comment (Dhravya) [inline apps/web/wrangler.toml]: Please never change this file

Comment (CodeTorso) [reply apps/web/wrangler.toml]: Oh sorry, accidentally committed this file

File: apps/web/app/(dash)/dynamicisland.tsx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                  transition: { duration: 0.15 },
+                }}
+                exit={{
+                  opacity: 0,
+                  transition: { duration: 0.15, delay: 0.2 },
+                }}
+              />
+            )}
+          </AnimatePresence>
+          <h3 className="text-[#858B92] z-50 relative">{item}</h3>
```
Comment (Dhravya) [inline apps/web/app/(dash)/dynamicisland.tsx:199]: don't use any arbitrary values with #color like this, 
either add a colour to the theme, or use an existing one. This kinda becomes hard to maintain

Comment (CodeTorso) [reply apps/web/app/(dash)/dynamicisland.tsx:199]: A lot of these changes are present with me locally , I will probably merge that later this day, I had been working locally for like 2 weeks without you knowing any changes, so I pushed that code just to make sure everyone gets to know the current state

Comment (Dhravya) [reply apps/web/app/(dash)/dynamicisland.tsx:199]: that's perfectly fine!

File: apps/web/app/(dash)/chat/chatWindow.tsx [hunk 3]
```diff
@@ -6,46 +6,234 @@ import QueryInput from "../home/queryinput";
...
+
+    if (!sourcesParsed.success) {
+      console.log(sources);
+      console.error(sourcesParsed.error);
+      toast.error("Something went wrong while getting the sources");
+      return;
+    }
+
+    console.log(sourcesParsed.data.ids);
+    console.log(sourcesParsed.data.metadata);
```
Comment (Dhravya) [inline apps/web/app/(dash)/chat/chatWindow.tsx]: @CodeTorso see here to populate the sources on the page

**Pull request closed without merging**